htdocs/pages/forumphp2016/inscription.php

```diff
@@ -237,20 +256,20 @@
 
 $formulaire->addGroup($groupe, 'groupe_type_reglement', $translator->trans('Règlement'), '&nbsp;', false);
-$formulaire->addElement('static', 'note', '', $translator->trans('Ces informations concernent la personne ou la société qui sera facturée.').'<br/><br/>');
+//$formulaire->addElement('static', 'note', '', $translator->trans('Ces informations concernent la personne ou la société qui sera facturée.').'<br/><br/>');
 $formulaire->addElement('text', 'societe_facturation', $translator->trans('Société'), array('size' => 50, 'maxlength' => 100, 'class' => 'span7'));
-$formulaire->addElement('text', 'nom_facturation', $translator->trans('Nom'), array('size' => 30, 'maxlength' => 40, 'class' => 'span7'));
-$formulaire->addElement('text', 'prenom_facturation', $translator->trans('Prénom'), array('size' => 30, 'maxlength' => 40, 'class' => 'span7'));
-$formulaire->addElement('textarea', 'adresse_facturation', $translator->trans('Adresse'), array('cols' => 42, 'rows'      => 10, 'class' => 'span7'));
-$formulaire->addElement('text', 'code_postal_facturation', $translator->trans('Code postal'), array('size' =>  6, 'maxlength' => 10, 'class' => 'span7'));
-$formulaire->addElement('text', 'ville_facturation', $translator->trans('Ville'), array('size' => 30, 'maxlength' => 50, 'class' => 'span7'));
+$formulaire->addElement('text', 'nom_facturation', $translator->trans('Nom'), array('size' => 30, 'maxlength' => 40, 'class' => 'span7', 'required' => 'true'));
+$formulaire->addElement('text', 'prenom_facturation', $translator->trans('Prénom'), array('size' => 30, 'maxlength' => 40, 'class' => 'span7', 'required' => 'true'));
+$formulaire->addElement('textarea', 'adresse_facturation', $translator->trans('Adresse'), array('cols' => 42, 'rows'      => 10, 'class' => 'span7', 'required' => 'true'));
+$formulaire->addElement('text', 'code_postal_facturation', $translator->trans('Code postal'), array('size' =>  6, 'maxlength' => 10, 'class' => 'span7', 'required' => 'true'));
+$formulaire->addElement('text', 'ville_facturation', $translator->trans('Ville'), array('size' => 30, 'maxlength' => 50, 'class' => 'span7', 'required' => 'true'));
 $formulaire->addElement('select', 'id_pays_facturation', $translator->trans('Pays'), $pays->obtenirPays());
-$formulaire->addElement('text', 'email_facturation', $translator->trans('Email (facture)'), array('size' => 30, 'maxlength' => 100, 'class' => 'span7'));
+$formulaire->addElement('text', 'email_facturation', $translator->trans('Email (facture)'), array('size' => 30, 'maxlength' => 100, 'class' => 'span7', 'required' => 'true'))->setType('email');
 
 
 if ($config_forum['coupons'] != []) {
     $formulaire->addElement('text', 'coupon',
-        $translator->trans('Coupon (en cas de soucis avec votre coupon, vous pouvez contacter le trésorier sur tresorier[at]afup.org)'),
+        $translator->trans('Bon de réduction'),
```

Comment:
:+1:

Jacques Bodin-Hullin

-- 
Monsieur Biz <https://monsieurbiz.com>
@jacquesbh <https://twitter.com/jacquesbh>
+33 (0)6 75 54 11 97
GPG : https://keybase.io/jacquesbh

Le lun. 1 août 2016 à 19:12, Xavier Leune <notifications@github.com> a
écrit :

> In htdocs/pages/forumphp2016/inscription.php
> <https://github.com/afup/web/pull/209#discussion_r73015629>:
>
> >
> >
> >  if ($config_forum['coupons'] != []) {
> >      $formulaire->addElement('text', 'coupon',
> > -        $translator->trans('Coupon (en cas de soucis avec votre coupon, vous pouvez contacter le trésorier sur tresorier[at]afup.org)'),
> > +        $translator->trans('Bon de réduction'),
>
> Je trouve que ça n'apporte pas grand chose mais que ça alourdi la lecture.
> Pour moi le seul soucis qu'on ait rencontré c'était des coupons appliqués
> en trop. Si tu veux on peut la conserver mais indiquée différemment et
> placée ailleurs
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/afup/web/pull/209/files/cf275cee2a948548c57458de5a89f1cc73480688#r73015629>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA0Z8wqogpS6Egp_q9k5SHTzgSKjSsRCks5qbikLgaJpZM4JZZjW>
> .
>